src/layouts/Layout.astro

```diff
@@ -425,69 +306,71 @@ function isPathActive(path) {
       });
     </script>
     <script async src="https://cdn.announcekit.app/widget-v2.js"></script>
-  </body>
-</html>
 
-<style>
```

Comment:
Inline `<style>` and `<script>` code is bundled by astro, so they don't appear on the page as is. I think the idea was to keep it at the very end for clarity, so I would keep that :)